bin/ocFunctions.inc

```diff
@@ -1788,17 +1788,69 @@ generateSeed() {
 function getSecret() {
   (
     resourceName=${1}
-    key=${2}
+    keys=${2}
     projectName=${3:-$(getProjectName)}
-    if [ -z "${resourceName}" ] || [ -z "${key}" ]; then
+    if [ -z "${resourceName}" ] || [ -z "${keys}" ]; then
```

Comment:
You may want to update the error message to say `keys`.

Reply:
get secret reverted to origional